test/scripts/install-provisioner-working.sh

```diff
@@ -28,16 +28,16 @@ if ! kind::cluster::exists "$CLUSTERNAME" ; then
     exit 2
 fi
 
-docker rmi ghcr.io/projectcontour/contour:latest --force
+docker rmi ghcr.io/projectcontour/contour:dev --force
 
 # Build the Contour Provisioner image.
-make -C ${REPO} container IMAGE=ghcr.io/projectcontour/contour VERSION=latest
+make -C ${REPO} container IMAGE=ghcr.io/projectcontour/contour VERSION=dev
 
 # Push the Contour Provisioner image into the cluster.
-kind::cluster::load::docker ghcr.io/projectcontour/contour:latest
+kind::cluster::load::docker ghcr.io/projectcontour/contour:dev
 
 # Install the Gateway provisioner using the loaded image.
-export CONTOUR_IMG=ghcr.io/projectcontour/contour:latest
+export CONTOUR_IMG=ghcr.io/projectcontour/contour:dev
```

Comment:
I think there's an issue here where even though the provisioner itself will be updated to use the newly built `-dev` image, any Contour instances managed by it don't get updated, since the tag hasn't changed, which can be problematic for the dev workflow.

Reply:
In that case, maybe an old pseudo-random image tag schema would be nice with some changes like we can add the prefix `dev` to it so that we can remove all the images starting with the `dev` tag.